### PR TITLE
Create chart for presto.

### PR DESCRIPTION
**Pull request by MarvinCai**
Fixes https://github.com/streamnative/charts/issues/458

### Motivation/Modifications
Create a separate helm chart for presto for chart modularization work.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation

- [x] `no-need-doc`

### CODE REVIEW

File: charts/sn-platform-stack/values.yaml [hunk 1]
```diff
@@ -18,9 +18,9 @@
 #
 
 global:
-  stackName: ""
-  stackVersion: ""
-
+  stackName: "<stack-name>"
```
Comment (fantapsody) [inline charts/sn-platform-stack/values.yaml:21]: Is this really needed to be defined?

File: charts/sn-platform-stack/values.yaml [hunk 2]
```diff
@@ -76,7 +77,7 @@ sn-console:
       
 sn-platform-vault:
   enabled: true
-  fullnameOverride: "<release name>-sn-platform"
+  fullnameOverride: "<stack-name>-sn-platform"
```
Comment (fantapsody) [inline charts/sn-platform-stack/values.yaml:80]: Can the default value be set to `""`?

File: charts/sn-platform-stack/Chart.yaml [hunk 3]
```diff
@@ -32,8 +32,8 @@ maintainers:
     email: support@streamnative.io
 dependencies: 
   - name: sn-platform
-    version: "1.4.0"
-    repository: https://charts.streamnative.io
+    version: "1.5.0"
+    repository: file://../sn-platform
```
Comment (fantapsody) [inline charts/sn-platform-stack/Chart.yaml:36]: Would this work when this chart is published to remote?

**Comment by ericsyh** [comment]
@MarvinCai Can u check with Yang Yang's review comment?

**Comment by ericsyh** [comment]
@FushuWang @maxsxu @ciiiii PTAL

File: charts/sn-platform-presto/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+apiVersion: v2
+description: StreamNative Platform Presto
+name: sn-platform-presto
+type: application
+version: 0.1.0
+appVersion: "0.1.0"
+home: https://streamnative.io
+sources:
+  - https://github.com/streamnative/charts/tree/master/charts/sn-platform-presto
+icon: https://streamnative.io/static/images/platform/sn-platform-header-element.svg
```
Comment (maxsxu) [inline charts/sn-platform-presto/Chart.yaml:29]: icon link seems broken

**Review by maxsxu — COMMENTED** [review]
Do we need to sync again from `charts/sn-platform/templates/presto`? 
As some changes were made against upstream after this PR.

**Comment by ericsyh** [comment]
> Do we need to sync again from `charts/sn-platform/templates/presto`? As some changes were made against upstream after this PR.

I think @ciiiii added Istio support for Presto and will plan to add the JMX exporter support. After these merged into the `charts/sn-platform/templates/presto` we should sync again.

**Comment by ericsyh** [comment]
Change plan and no more dev on it.

**Pull request closed without merging**